protos/protos.go

```diff
@@ -126,7 +126,7 @@ func (p Protocol) String() string {
 }
 
 type Protocols interface {
-	BpfFilter(with_vlans bool) string
+	BpfFilter(with_vlans bool, with_icmp bool) string
```

Comment:
No, we also plan to go with camelCase in the config. See here for an example: https://github.com/elastic/filebeat/blob/master/etc/filebeat.yml#L43